### ISSUE TITLE
More info on the dataset used and finetuning results 

### ISSUE THREAD

**Issue by SharmaSubham975** [dino-chiio/blip-vqa-finetune#1]
Can you please share more info on the dataset used for training and some of the results

**Comment by dino-chiio** [comment]
I have uploaded the data [here](https://drive.google.com/file/d/1tt6qJbOgevyPpfkylXpKYy-KaT4_aCYZ/view?usp=sharing)

The test set does not have a label because I evaluated it on the Kaggle competition of my course. The accuracy I got was 96%.

**Comment by uttam-scai** [comment]
Thank you for your dataset. I downloaded your data and i have one question - Have you created one folder for each image? or does the gap (folder 1, 3, 33) in the dataset means I can keep many images of same type in the same folder? I have added a screenshot for clear understanding. Thank you !!!

![image](https://github.com/dino-chiio/blip-vqa-finetune/assets/54339614/cd85f42f-578d-46ab-92bd-e4107d503e07)

**Comment by dino-chiio** [comment]
> Thank you for your dataset. I downloaded your data and i have one question - Have you created one folder for each image? or does the gap (folder 1, 3, 33) in the dataset means I can keep many images of same type in the same folder? I have added a screenshot for clear understanding. Thank you !!!
> 
> ![image](https://private-user-images.githubusercontent.com/54339614/301891627-cd85f42f-578d-46ab-92bd-e4107d503e07.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY5NjY0MjYsIm5iZiI6MTcwNjk2NjEyNiwicGF0aCI6Ii81NDMzOTYxNC8zMDE4OTE2MjctY2Q4NWY0MmYtNTc4ZC00NmFiLTkyYmQtZTQxMDdkNTAzZTA3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjAzVDEzMTUyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ0ODgxNWJlMjg1MzA0Mjc1MzZjNTg2YThjYmIyNDYxZDRiMjI4NTI5YzJlZWM4ZGZhMTVmYTgwZmYzZGVjMWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.sKaMTg38nIcC3u3Fpdb4f9auKk7gDtK9bU3Gf6fEJTw)

Each folder (number) contains only one image. It means that it is one data sample. You can see the training data information stored in the .json file. It is loaded in the code.
